Include standard plugins because z3c.autoinclude does not work without an 
installed schooltool/__init__.py

Remove dependency.
Index: schooltool/src/schooltool/standard/configure.zcml
===================================================================
--- schooltool.orig/src/schooltool/standard/configure.zcml
+++ schooltool/src/schooltool/standard/configure.zcml
@@ -7,8 +7,9 @@
 
   <include package="schooltool.common" />
 
-  <include package="z3c.autoinclude" file="meta.zcml" />
-  <includePlugins package="schooltool" file="plugin.zcml" />
+  <include package="schooltool.lyceum.journal" />
+  <include package="schooltool.gradebook" />
+  <include package="schooltool.intervention" />
 
   <include package="schooltool.help" />
   <include package="schooltool.timetable" />
Index: schooltool/setup.py
===================================================================
--- schooltool.orig/setup.py
+++ schooltool/setup.py
@@ -67,7 +67,6 @@ Javascript will be usable, although perh
                       'setuptools',
                       'xlrd',
                       'xlwt',
-                      'z3c.autoinclude',
                       'z3c.form>=2.0',
                       'z3c.formui',
                       'z3c.macro',
Index: schooltool/src/schooltool.egg-info/requires.txt
===================================================================
--- schooltool.orig/src/schooltool.egg-info/requires.txt
+++ schooltool/src/schooltool.egg-info/requires.txt
@@ -8,7 +8,6 @@ rwproperty
 setuptools
 xlrd
 xlwt
-z3c.autoinclude
 z3c.form>=2.0
 z3c.formui
 z3c.macro
